clang/include/clang/AST/VTableBuilder.h

```diff
@@ -246,17 +246,17 @@ class VTableLayout {
   // point for a given vtable index.
   typedef llvm::SmallVector<unsigned, 4> AddressPointsIndexMapTy;
 
+  using VTableIndicesTy = llvm::SmallVector<std::size_t>;
+
 private:
-  // Stores the component indices of the first component of each virtual table in
-  // the virtual table group. To save a little memory in the common case where
-  // the vtable group contains a single vtable, an empty vector here represents
-  // the vector {0}.
-  OwningArrayRef<size_t> VTableIndices;
+  // Stores the component indices of the first component of each virtual table
+  // in the virtual table group.
+  VTableIndicesTy VTableIndices;
 
-  OwningArrayRef<VTableComponent> VTableComponents;
+  std::vector<VTableComponent> VTableComponents;
```

Comment:
Why not SmallVector?

Reply:
`std::vector` is the most similar to the old code in terms of performance characteristics. `SmallVector` is probably fine here, though, it's just that my goal wasn't to optimize `VTableBuilder` so much as remove the type so I was going for minimal change in this PR.

Reply:
Our programmers manual recommends SmallVector over std::vector. We could use `SmallVector<T, 0>` if having a small buffer on the stack doesn't make sense.
https://llvm.org/docs/ProgrammersManual.html#llvm-adt-smallvector-h